corehq/motech/repeaters/expression/forms.py

```diff
@@ -18,9 +18,34 @@ class CaseExpressionRepeaterForm(GenericRepeaterForm):
         help_text=_("Items to add to the end of the URL. Please see the documentation for more information.")
     )
 
+    update_case_filter_expression = JsonField(
```

Comment:
will this always be an update?
We are limiting to allowing updates to the case that was sent?
Should extend this to creating new cases or updating related cases as well? We could rename this to `case_action_filter_expression` and `case_action_expression` for the action to be taken?

Reply:
no, it could be any case operation. I renamed this field a number of times and then gave up but `case_action_X` seems reasonable. I can update.

Reply:
66c50f85eb0ed930d2886eb48c740c1bb16606e1